src/Microsoft.Extensions.OData.Migration/Microsoft.Extensions.OData.Migration.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>0.9.2-preview</Version>
```

Comment:
>0.9.2-preview [](start = 13, length = 13)

the version is by design?

Reply:
No, it's used for internal consumption in Intune. I will change it to 1.0.0 unless you have other recommendations.